apps/whale-api/src/module.api/poolswap.pathfinding.service.ts

```diff
@@ -154,6 +158,16 @@ export class PoolSwapPathFindingService {
             ab: formatNumber(new BigNumber(poolPair['reserveA/reserveB'])),
             ba: formatNumber(new BigNumber(poolPair['reserveB/reserveA']))
           }
+        }
+
+        const estimatedReturnTokenB = computeReturnInDestinationToken([swapPathPoolPair], tokenA, estimatedReturnTokenA)
+        const swapPathWithDirection = getSwapPathWithDirection(swapPathPoolPair, tokenA, tokenB, estimatedReturnTokenA, estimatedReturnTokenB)
+        const poolpairDexFees = await this.poolPairFeesServices.getDexFees(swapPathWithDirection)
+        estimatedReturnTokenA = estimatedReturnTokenB
+        poolPairs.push({
+          ...swapPathPoolPair,
+          estimatedReturn: estimatedReturnTokenB.toFixed(8),
+          estimatedDexFees: poolpairDexFees
```

Comment:
Noticed that `estimatedReturn` here is just a duplication of `priceRatio.ba`.
I suggest the following:

```typescript
{
  priceRatio: { ab: '...', ba: '...' },
  dexFees: { ab: '...', ba: '...' }
}
```

Reply:
@eli-lim ab and ba kinda denotes that it is a path, I think it more makes sense to use
``` 
estimatedDexFeesInPct: {
          tokenA: '0.00300000',
          tokenB: '0.00000000'
        },
```